app/lib/default_error_messages.rb

```diff
@@ -1,46 +1,55 @@
 class DefaultErrorMessages
   def self.generate!(service_type: "unfilled")
-    EfileError.find_or_create_by!(
-      code: "BUNDLE-FAIL",
-      message: "An error occurred while bundling the submission XML.",
-      source: :internal,
-      expose: false,
-      service_type: service_type
-    )
     EfileError.find_or_create_by!(
       code: "USPS-2147219401",
       message: "Address Not Found.",
       source: :usps,
       expose: true
     )
-    EfileError.find_or_create_by!(
-      code: "PDF-1040-FAIL",
-      message: "Could not generate IRS Form 1040 PDF.",
-      source: :internal,
-      expose: false,
-      service_type: service_type
-    )
-    EfileError.find_or_create_by!(
-      code: "TRANSMISSION-SERVICE",
-      message: "Error communicating with GYR Efiler service.",
-      source: :internal,
-      expose: false,
-      service_type: service_type
-    )
-    EfileError.find_or_create_by!(
-      code: "TRANSMISSION-RESPONSE",
-      message: "Unexpected transmission response format from IRS.",
-      source: :internal,
-      expose: false,
-      service_type: service_type
-    )
-    EfileError.find_or_create_by!(
-      code: "BANK-DETAILS",
-      message: "Some of the provided bank account details are incorrect or absent.",
-      source: :intake,
-      expose: true,
-      auto_wait: true,
-      service_type: service_type
-    )
+
+    service_types = case service_type
+                    when :state_file
```

Comment:
coming back to this after finishing the rest of the review - i think we should remove the default parameter value to this method, but let me know if you think otherwise!

Reply:
well, there are a few tests that call `generate!` with no argument, but i can change them

Reply:
your call - i think it'd be good to either change the default param value from `"unfilled"` to `:unfilled`, or to change the calls that don't provide a service_type, but i don't think either is strictly necessary, and it's definitely not part of the scope of this pr per se